src/System.Security.Cryptography.Cng/pkg/System.Security.Cryptography.Cng.pkgproj

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <ProjectReference Include="..\ref\System.Security.Cryptography.Cng.csproj">
+      <SupportedFramework>net461;netcoreapp2.0;uap10.1</SupportedFramework>
+    </ProjectReference>
+    <ProjectReference Include="..\src\System.Security.Cryptography.Cng.csproj" />
+    
+    <!-- All elements from previous packages that will be included in the newly built package -->
+    <HarvestIncludePaths Include="ref/net46;lib/net46;runtimes/win/lib/net46" />
+    <HarvestIncludePaths Include="ref/netstandard1.3" />
```

Comment:
It looks like for Servicecontroller we're repackaging the Unix netstandard1.5 lib assembly as a netstandard1.4 ridless assembly. I don't see how that doesn't cause issues since the assembly versions between the netstandard1.4 ref and lib would differ.

It seems like I'll need to build a PNSE lib assembly for netstandard1.3 to make the packaging verification happy. Sort of like what is happening in [TypeExtensions](https://github.com/dotnet/corefx/blob/af74a86c3003d487e981655283488f8b03829b29/src/System.Reflection.TypeExtensions/src/System.Reflection.TypeExtensions.csproj#L13). But then there's the issue that the targets would need an older version of the ref cs file from which to produce the PSNE assembly... unless PSNE generation looks at the already produced dll.